Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -878,3 +878,237 @@
+
+- type: reagent
+  id: Fomepizole
+  name: reagent-name-fomepizole
+  group: Medicine
+  desc: reagent-desc-fomepizole
+  physicalDesc: reagent-physical-desc-odorless
+  flavor: medicine
+  color: "#afdd9a"
+  metabolisms:
+    Medicine:
+      effects:
+      - !type:GenericStatusEffect
+        key: Jitter
+        time: 10.0
+        type: Remove
+      - !type:GenericStatusEffect
+        key: Stutter
+        time: 10.0
+        type: Remove
+      - !type:GenericStatusEffect
+        key: SeeingRainbows
+        time: 10.0
+        type: Remove
+
+- type: reagent
+  id: Haloperidol
+  name: reagent-name-haloperidol
+  group: Medicine
+  desc: reagent-desc-haloperidol
+  physicalDesc: reagent-physical-desc-odorless
+  flavor: medicine
+  color: "#ded0ee"
+  metabolisms:
+    Medicine:
+      effects:
+      - !type:HealthChange
+        probability: 0.2
+        damage:
+          types:
+            Cellular: 1
+      - !type:AdjustReagent
+        reagent: Epinephrine
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Epinephrine
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:AdjustReagent
+        reagent: Ephedrine
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Ephedrine
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:AdjustReagent
+        reagent: Stimulants
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Stimulants
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:AdjustReagent
+        reagent: THC
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: THC
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:AdjustReagent
+        reagent: SpaceDrugs
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: SpaceDrugs
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:AdjustReagent
+        reagent: Desoxyephedrine
+        amount: -2
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Desoxyephedrine
+          min: 1
+        reagent: Haloperidol
+        amount: -1
+      - !type:GenericStatusEffect
+        key: Jitter
+        time: 5.0
+        type: Remove
+      - !type:GenericStatusEffect
+        key: Stutter
+        time: 5.0
+        type: Remove
+      - !type:GenericStatusEffect
+        key: SeeingRainbows
+        time: 5.0
+        type: Remove
+      - !type:GenericStatusEffect
+        conditions:
+        - !type:ReagentThreshold
+        probability: 0.5
+        key: ForcedSleep
+        component: ForcedSleeping
+        refresh: true
```

Comment:
wut.

This seems unbalanceable when it just gives out 6 reagents and 20 effects.

Reply:
> wut.
> 
> This seems unbalanceable when it just gives out 6 reagents and 20 effects.

I will remove the stimulants, but otherwise - it removes "simple" narcotics and removes the concomitant narcotic intoxication. Also, if you were injected with a drug in battle, you can't just drink haloperidol and continue fighting, since there is a 50% chance that you will fall asleep during metabolism. Also, with a 20% chance, it will cause cellular damage - I agree, weakly, perhaps it is worth adding additional impact as a counteraction to a wide effect.
In fact, its effect corresponds to the effect from reality (except for the treatment of schizophrenia and the like, for lack of it).
Also, there are no medicine that could remove such simple narcotics from the body - I'm not talking about serious ones, with complex creation and deadly effects, which the syndicate uses.
It may be worth complicating the recipe, but such medicine are definitely needed - it is very unpleasant to walk the whole shift with a rainbow in your eyes because of the frezon leak.